main.go

```diff
@@ -454,10 +457,13 @@ func getQANClient(ctx context.Context, sqlDB *sql.DB, dbName, qanAPIAddr string)
 	return qan.NewClient(conn, db)
 }
 
-func getDBaaSControllerClient(ctx context.Context, dbaasControllerAPIAddr string) *dbaas.Client {
+func getDBaaSControllerClient(ctx context.Context, dbaasControllerAPIAddr string, settings *models.Settings) *dbaas.Client {
+	if !settings.DBaaS.Enabled {
+		return dbaas.NewClient(nil)
+	}
 	opts := []grpc.DialOption{
 		grpc.WithInsecure(),
-		grpc.WithConnectParams(grpc.ConnectParams{Backoff: backoff.Config{MaxDelay: 2 * time.Second}, MinConnectTimeout: 10 * time.Second}),
+		grpc.WithConnectParams(grpc.ConnectParams{Backoff: backoff.Config{MaxDelay: 10 * time.Second}, MinConnectTimeout: 10 * time.Second}),
```

Comment:
Do we really need to override the default client connection params (`grpc.WithConnectParams`) to the local server here?
https://github.com/grpc/grpc-go/blob/7b167fd6eca1ab8f05ec14085d63197cacd41438/dialoptions.go#L236-L255

This looks like a suboptimal backoff strategy.

The issue with high CPU usage can appear again when dbaas-controller with any unpredictable reason fail - this can lead to fails pmm-managed and other components.

Reply:
yeah, it also can happen if qan fails and any other component. but supervisord should restart them if they fails. If they kepp failing, then something wrong.